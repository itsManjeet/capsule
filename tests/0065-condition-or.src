condition_1 := fun (v) {
    println ("CONDITION 1");
    return v;
};

condition_2 := fun (v) {
    println ("CONDITION 2");
    return v;
};

if condition_1(false) or condition_2(true) {
    println ("I MUST RUN");
}

if condition_1(true) or condition_2(true) {
    println ("I MUST RUN");
}

if condition_1(true) or condition_2(false) {
    println ("I MUST RUN");
}