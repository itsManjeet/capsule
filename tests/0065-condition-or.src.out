CONDITION 1
CONDITION 2
I MUST RUN
CONDITION 1
I MUST RUN
CONDITION 1
I MUST RUN
